app/ui.js

```diff
@@ -98,6 +98,8 @@ var UI;
         lastKeyboardinput: null,
         defaultKeyboardinputLen: 100,
 
+        reconnect_enabled: false,
```

Comment:
Yeah, I find it tough to find a good name here as well. Your suggestion works fine for me